Protocol.md

```diff
@@ -432,8 +434,8 @@ Frame Contents
 * __Frame Type__: (6 bits) 0x02 
 * __Flags__: (10 bits)
      * (__M__)etadata: Metadata present
-* __Time-To-Live (TTL)__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer of Time (in milliseconds) for validity of LEASE from time of reception. Value MUST be > 0. 
-* __Number of Requests__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer of Number of Requests that may be sent until next LEASE. Value MUST be > 0. 
+* __Time-To-Live (TTL)__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer of Time (in milliseconds) for validity of LEASE from time of reception. Value MUST be >= 0. 
```

Comment:
Why allowing 0 for the duration of the lease?

Reply:
I believe it was a typo in one or another place. But if you look at lease description (https://github.com/rsocket/rsocket/blob/master/Protocol.md#lease-frame-0x02) you will see the sequence of the following words:

```
A Responder implementation MAY stop all further requests by sending a LEASE with a value of 0 for Number of Requests or Time-To-Live.
```

From the above, both TTL and Number of Requests can be 0 to prevent all further requests. 
I agree that TTL 0 makes no sense.

Reply:
fixed